PWGHF/Core/SelectorCuts.h

```diff
@@ -479,17 +479,17 @@ constexpr double binsPt[nBinsPt + 1] = {
   36.};
 auto vecBinsPt = std::vector<double>{binsPt, binsPt + nBinsPt + 1};
 
-// default values for the cuts
-constexpr double cuts[nBinsPt][nCutVars] = {{0.400, 0.4, 0.4, 0.4, 0., 0.005, 0., -1.},  /* 0  < pT < 1  */
-                                            {0.400, 0.4, 0.4, 0.4, 0., 0.005, 0., -1.},  /* 1  < pT < 2  */
-                                            {0.400, 0.4, 0.4, 0.4, 0., 0.005, 0., -1.},  /* 2  < pT < 3  */
-                                            {0.400, 0.4, 0.4, 0.4, 0., 0.005, 0., -1.},  /* 3  < pT < 4  */
-                                            {0.400, 0.4, 0.4, 0.4, 0., 0.005, 0., -1.},  /* 4  < pT < 5  */
-                                            {0.400, 0.4, 0.4, 0.4, 0., 0.005, 0., -1.},  /* 5  < pT < 6  */
-                                            {0.400, 0.4, 0.4, 0.4, 0., 0.005, 0., -1.},  /* 6  < pT < 8  */
-                                            {0.400, 0.4, 0.4, 0.4, 0., 0.005, 0., -1.},  /* 8  < pT < 12 */
-                                            {0.400, 0.4, 0.4, 0.4, 0., 0.005, 0., -1.},  /* 12 < pT < 24 */
-                                            {0.400, 0.4, 0.4, 0.4, 0., 0.005, 0., -1.}}; /* 24 < pT < 36 */
+// default values for the cuts                m,  ptP, ptK, ptPi, chi2PCA, dL, cosp, dLXY, NdLXY, ct, ImpParXY mass(Kpi)
+constexpr double cuts[nBinsPt][nCutVars] = {{0.4, 0.4, 0.4, 0.4, 0., 0.005, 0., 0.005, 1., 100., 0.5, -1.},  /* 0  < pT < 1  */
+                                            {0.4, 0.4, 0.4, 0.4, 0., 0.005, 0., 0.005, 1., 100., 0.5, -1.},  /* 1  < pT < 2  */
+                                            {0.4, 0.4, 0.4, 0.4, 0., 0.005, 0., 0.005, 1., 100., 0.5, -1.},  /* 2  < pT < 3  */
+                                            {0.4, 0.4, 0.4, 0.4, 0., 0.005, 0., 0.005, 1., 100., 0.5, -1.},  /* 3  < pT < 4  */
+                                            {0.4, 0.4, 0.4, 0.4, 0., 0.005, 0., 0.005, 1., 100., 0.5, -1.},  /* 4  < pT < 5  */
+                                            {0.4, 0.4, 0.4, 0.4, 0., 0.005, 0., 0.005, 1., 100., 0.5, -1.},  /* 5  < pT < 6  */
+                                            {0.4, 0.4, 0.4, 0.4, 0., 0.005, 0., 0.005, 1., 100., 0.5, -1.},  /* 6  < pT < 8  */
+                                            {0.4, 0.4, 0.4, 0.4, 0., 0.005, 0., 0.005, 1., 100., 0.5, -1.},  /* 8  < pT < 12 */
+                                            {0.4, 0.4, 0.4, 0.4, 0., 0.005, 0., 0.005, 1., 100., 0.5, -1.},  /* 12 < pT < 24 */
+                                            {0.4, 0.4, 0.4, 0.4, 0., 0.005, 0., 0.005, 1., 100., 0.5, -1.}}; /* 24 < pT < 36 */
```

Comment:
thanks @phymanshu , ok for me to add such cuts if needed, however I'd use default values such that no new selections are applied (i.e.: I'd use `0` as default for `decLengthXY` and `normDecLXY`, and `1e+10` for `ct` and `impParXY`)